examples/contrastive/contrastive.cpp

```diff
@@ -178,7 +179,6 @@ int main(int argc, char ** argv) {
             }
 
             LOG_TEE("%s", llama_token_to_piece(ctx_expert, new_token_id_expert).c_str());
-            fflush(stdout);
```

Comment:
You _do_ want to flush stdout here, as stdout is otherwise not flushed until a newline is printed.